notebooks/04.02-Simple-Scatter-Plots.ipynb

```diff
@@ -37,10 +37,8 @@
   },
```

Comment:
Link it away! <br/> _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/1/' state='OPEN' cell_index=22>ReviewNB</a>_